RM_SUI/RM_SUI/StoreTabBar/Tabs/ProductsView.swift

```diff
@@ -0,0 +1,145 @@
+                        .foregroundColor(.white)
+                    Label(Constants.searchLabelName, systemImage: Constants.searchLabelImageName)
+                        .font(.title)
+                        .foregroundColor(.indigo)
+                        .offset(CGSize(width: -40, height: 0))
+                }
+                NavigationLink {
+                    FilterView()
+                } label: {
+                    Image(systemName: Constants.filterButtonImageName)
+                }
+                .offset(CGSize(width: -30, height: 0))
+                .font(.largeTitle)
+            }
+        }
+        .ignoresSafeArea()
+        .frame(maxHeight: 80)
+    }
+
+    private func furnitureItemView(product: Product) -> some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 20)
+                .foregroundColor(Color(.systemGray5))
+            HStack {
+                Image(systemName: product.image)
+                    .resizable()
+                    .scaledToFit()
+                    .frame(width: 50, height: 50)
+                    .foregroundColor(.red)
+
+                Spacer()
+                VStack {
+                    Text(product.name)
+                        .font(.body)
+                        .fontWeight(.bold)
+                    HStack {
+                        Button {
+                            productsViewModel.changeSellCountFor(
+                                value: 1,
+                                currentProduct: product
+                            )
+                        } label: {
+                            Text("+")
+                        }
+                        Text("\(product.sellCount)")
+                            .frame(
+                                width: 35,
+                                height: 35
+                            )
+                            .overlay {
+                                RoundedRectangle(cornerRadius: 5)
+                                    .stroke(Color.gray, lineWidth: 4)
+                            }
+                        Button {
+                            productsViewModel.changeSellCountFor(
+                                value: -1,
+                                currentProduct: product
+                            )
+                        } label: {
+                            Text("-")
+                        }
+                    }
+                }
+                .font(.title)
+                Spacer()
+                VStack {
+                    Text(String(product.price))
+                        .strikethrough()
+                        .foregroundColor(Color(.systemGray))
+                        .font(.title3)
+                    Text("$\(String(product.discountPrice))")
+                        .foregroundColor(.indigo)
+                        .font(.title)
+                }
+            }
+            .padding()
+        }
+    }
+}
+
+// MARK: - Preview
```

Comment:
```suggestion
///
```
У тебя где-то /// Описание где-то // MARK: